tests/python/TestCommand.py

```diff
@@ -41,6 +41,26 @@ def __init__(self, *args, **kwargs):
         self.hostname = "localhost"
         self.port = 55557
 
+        db_up = False
+        attempts = 0
+        while(not db_up):
+            try:
+                db = vdms.vdms()
+                db.connect(self.hostname, self.port)
+                db.disconnect()
+                db_up = True
+                if (attempts > 0):
+                    print("Connection to VDMS successful.")
+            except:
+                print("Attempt", attempts,
+                      "to connect to VDMS failed, retying...")
+                attempts += 1
+                time.sleep(1) # sleeps 1 second
```

Comment:
Does this commit belong in this branch or is this showing up cause GitHub is messing it up.

Reply:
are you asking this commit to be in a separate PR? We can do that, sure. I will split the PR and go ahead with the merge.